docker/genericsql/Dockerfile

```diff
@@ -0,0 +1,13 @@
+
+FROM demisto/python3:3.8.1.6442
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git \
+  && apk add gcc libc-dev g++ libffi-dev libxml2 unixodbc-dev mariadb-dev postgresql-dev \
+  && apk add freetds freetds-dev pkgconfig\
+  && pip install pyodbc \
```

Comment:
Added to the Pipfile, also has to be here because it needs to be installed after "freetds-dev", when I checked it again freetds and pkgconfig are not necessary for this docker.